### PR TITLE
Create reference-architecture-css.md

### PR DESCRIPTION
**Pull request by joealewine**
### Description

Add reference architecture doc to CSS DA repo.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by SirSpidey — APPROVED** [review]
So you can get this out, I didn't do a re-review of the earlier content in landing zone

**Review by SirSpidey — DISMISSED** [review]

File: reference-architecture/reference-architecture-css.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+- Storage: Backup, Archive
+- Networking: Cloud native contectivity
+- Security: Data Security, Identity & Access, Application Security, Threat Detection and Response, Infrastructure & Endpoints, Governance, Risk & Compliance
+- Resiliency: High Availability
+- Service Management: Monitoring, Logging, Auditing / tracking, Automated Deployment
+
+<br>
+
+![heatmap](heat-map-ccs.svg "Current diagram"){: caption="Figure 2. Architecture design scope" caption-side="bottom"}
```
Comment (bhakta-ibm) [inline reference-architecture/reference-architecture-css.md:111]: You didn't include the heatmap image as part of PR, it was included in my zip.

**Comment by SirSpidey** [comment]
@joealewine Another issue I missed: directory needs to be plural: `/reference architectures/`

**Review by SirSpidey — APPROVED** [review]
Things look like they're in the right places now. 👍

**Comment by SirSpidey** [comment]
@joealewine After you resolve the conflict with the `core-security-services-architecture.svg` file, we can probably get it merged.

**Review by SirSpidey — DISMISSED** [review]
🐎

**Review by SirSpidey — DISMISSED** [review]
re-approving

**Review by bhakta-ibm — DISMISSED** [review]
lgtm

**Comment by bhakta-ibm** [comment]
@ocofaigh can you please merge this?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@bhakta-ibm The diagram was recently updated to include COS bucket support for event notifications - it seems the diagram in this PR does not have that

**Comment by bhakta-ibm** [comment]
The PR should just have reference to that diagram should not replace it. So not sure the diagram is commited.

**Comment by SirSpidey** [comment]
> The PR should just have reference to that diagram should not replace it. So not sure the diagram is commited.

@bhakta-ibm Just heard from Conall, so I think you are working on this. But the background is that the directory needs to be plural for the cloud docs jobs (`reference architectures`), so the new diagram probably needs to be renamed/added to the PR.

**Comment by bhakta-ibm** [comment]
> > The PR should just have reference to that diagram should not replace it. So not sure the diagram is commited.
> 
> @bhakta-ibm Just heard from Conall, so I think you are working on this. But the background is that the directory needs to be plural for the cloud docs jobs (`reference architectures`), so the new diagram probably needs to be renamed/added to the PR.

@joealewine is on it, the branch is on his fork so couldn't make the change.

**Comment by joealewine** [comment]
@bhakta-ibm @ocofaigh Ok I think I've got it fixed now (or rather, just renamed, so the file is the same as it was in the main branch)

**Review by ocofaigh — APPROVED** [review]

**Review by ocofaigh — CHANGES_REQUESTED** [review]
You need to remove this file from the PR:
```
Forbid binaries...........................................................Failed
- hook id: forbid-binary
- exit code: 1

Error:  .DS_Store appears to be a binary file
```

Would you be able to add it to the .gitignore file to ensure it can't be committed in the future?

**Comment by SirSpidey** [comment]
> Would you be able to add it to the .gitignore file to ensure it can't be committed in the future?

@joealewine We have these instructions for ignoring .DS_STORE files: [Macs only: Ignoring .DS_Store files](https://test.cloud.ibm.com/docs-internal/writing?topic=writing-setting-up-your-markdown-environment#ignore-ds_store)

**Comment by joealewine** [comment]
I will not be adding it to gitignore. Every time I touch something in this PR something else seems to break. So I'll let one of you do that. I just want this thing merged so I can link to it from other docs. That's it

**Review by bhakta-ibm — APPROVED** [review]
lgtm

**Review by ocofaigh — APPROVED** [review]

**Comment by hvetsa** [comment]
> I will not be adding it to gitignore. Every time I touch something in this PR something else seems to break. So I'll let one of you do that. I just want this thing merged so I can link to it from other docs. That's it

I agree, we are holding a lot of updates for next revisions. Let us push these into repo and address next set of changes in another PR.

**Pull request merged**